src/Hng.Application/Features/Notifications/Commands/CreateNotificationSettingsCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Hng.Application.Features.Notifications.Dtos;
+using MediatR;
+
+namespace Hng.Application.Features.Notifications.Commands
+{
+    public class CreateNotificationSettingsCommand : IRequest<NotificationSettingsDto>
+    {
+        public CreateNotificationSettingsCommand(CreateNotificationSettingsDto notificationSettingsDto, string loggedInUserId)
+        {
+            NotificationBody = notificationSettingsDto;
+            LoggedInUserId = loggedInUserId;
```

Comment:
this would make you have to get logged in user from the controller and pass it into the command, use the GetLoggedInUserId method found in the authentication service class in hng.infrastructure.

Use it to get the logged in user in the command handler directly , instead of passing it from the controller